examples/pub_sub/Cargo.toml

```diff
@@ -7,5 +7,5 @@ edition = "2018"
 log = "0.4"
 web_logger = "0.1"
 serde = { version = "1.0", features = ["derive"] }
-yew = { path = "../.." }
+yew = { path = "../..", features = ["std_web"] }
```

Comment:
Actually `stdweb` isn't needed as an explicit dependency. My preferred fix would be to:
1. Remove `stdweb` dependency
2. Follow the other `stdweb`/`web-sys` agnostic examples and setup the `Cargo.toml` like this:

```toml
yew = { path = "../.." }

[features]
std_web = ["yew/std_web"]
web_sys = ["yew/web_sys"]
```

Reply:
I cannot find how to start an example that uses `web_sys`. Can you link to some instructions? :)

Reply:
@totorigolo you can also check out this build script (although it's kinda broken atm) https://github.com/yewstack/yew/blob/master/examples/build_all.sh